Sources/Generator/UIGenerator.swift

```diff
@@ -271,7 +363,14 @@ public class UIGenerator: Generator {
         if let field = constraint.field {
             constraintLine = "constraints.\(field) = \(constraintLine).constraint"
         }
-        return constraintLine
+
+        if let condition = constraint.condition {
+            l("if \(try! constraint.generateSwiftCondition(condition: condition, viewName: name))") {
```

Comment:
Generator should support exceptions now so the `try!` should not be needed.

Reply:
Wait, I thought I removed all of these pesky triers.